src/Workspaces/CoreTest/Execution/SnapshotSerializationTests.cs

```diff
@@ -699,21 +699,6 @@ private static AnalyzerFileReference CreateShadowCopiedAnalyzerReference(TempRoo
             return new AnalyzerFileReference(original, new MockShadowCopyAnalyzerAssemblyLoader(ImmutableDictionary<string, string>.Empty.Add(original, shadow.Path)));
         }
 
-        private interface INullLanguageService : ILanguageService { }
-
-        [ExportLanguageService(typeof(INullLanguageService), TestLanguage), Shared]
-        private class NullLanguageService : INullLanguageService
```

Comment:
👍 Nice, this wasn't correctly defined anyway (needed `[PartNotDiscoverable]`) so good on removing it